app/views/templates/_form.html.erb

```diff
@@ -12,6 +12,8 @@
     <li class="active"><a id="primary_tab" href="#primary" data-toggle="tab"><%= _("Template") %></a></li>
     <li><a href="#template_inputs" data-toggle="tab"><%= _("Inputs") %></a></li>
     <%= render "#{@type_name_plural}/custom_tab_headers" unless type == 'ptable'%>
+    <%= render_pagelets_for(:job_template_tab_headers) if type == 'job_template' %>
```

Comment:
How about having this as a generic extension point not tied to specific template type and then letting the pagelet itself decide if it wants to show its content or not based on the template for which it is being shown?

Reply:
And for which templates do we want to allow that?

Reply:
Most likely all of them. So this would be just `<%= render_pagelets_for(:tab_headers) %>`

Reply:
So you are talking about changing the labels inside the `render_pagelets_for`, and keeping the `if type == 'job_template' ` conditions? 
Or also remove the conditions from here and add them to the Pagelet instead?

Reply:
Sorry for the late reply, remove the conditions and add them to the pagelets instead